manifests/init.pp

```diff
@@ -427,6 +429,7 @@
   Optional[Integer[0]] $dhcp_mclt = $::foreman_proxy::params::dhcp_mclt,
   Optional[Integer[0, 255]] $dhcp_load_split = $::foreman_proxy::params::dhcp_load_split,
   Optional[Integer[0]] $dhcp_load_balance = $::foreman_proxy::params::dhcp_load_balance,
+  Variant[Array[String], Optional[String]] $dhcp_includes = $::foreman_proxy::params::dhcp_includes,
```

Comment:
Are these absolute paths or relative? I'd also limit it to just `Array[String]` and default it to `[]`